src/main/resources/explanations/challenge32_reason.adoc

```diff
@@ -0,0 +1,10 @@
+*Why you should be careful with AI (or ML) and secrets*
```

Comment:
Is it worth mentioning here that a big problem with LLM AI is that user input is implicitly trusted? The prompt it was given by its creator and the text the user is sending are weighted similarly and especially when a user can write lots of repeated text they can easily overwhelm the prompt that was given to do things like protect secrets or even just to tell it what to do.

Reply:
There simply is no separation between prompt and input. They are conceptually the same.